INSTALLATION.md

```diff
@@ -140,6 +140,19 @@ and create the VMs:
 ./45-add-vms.sh
 ```
 
+## Enable multi-NUMA VMs
+In the VM template itself you can only define a **pre-requisite for NUMA**. With the extra step below, you define the guest side NUMA topology with some random mapping to physical resources. Note that the random mapping may be non-optimal, but it needs to exist. Without this step, there is no NUMA enabled in the guests.
+
+```
+#!/bin/bash
+. common_funcs
+for j in $(seq 1 $NUMA_NODES); do
+  curl_api vms/$id/numanodes -X POST -d "<vm_numa_node><cpu><cores><core><index>$j</index></core></cores></cpu><index>$j</index><memory>$NUMA_MEMORY</memory></vm_numa_node>"
+    done
+done
+```
+This [creates a NUMA node]((http://ovirt.github.io/ovirt-engine-api-model/master/#services/vm_numa_nodes/methods/add)) for each vCPU.
+
```

Comment:
not sure it's worth quoting the code when it's integrated in 45-add-vms, rather just document the optional parameter